PWGHF/D2H/DataModel/ReducedDataModel.h

```diff
@@ -442,6 +504,26 @@ DECLARE_SOA_TABLE(HfRedBsDsMls, "AOD", "HFREDBSDSML", //! Table with ML scores f
 
 using HfRedCandBs = soa::Join<HfCandBsExt, HfRedBsProngs>;
 
+namespace hf_cand_lb_reduced
+{
+DECLARE_SOA_INDEX_COLUMN_FULL(Prong0Lc, prong0Lc, int, HfRed3Prongs, "_0");          //! Prong0 index
+DECLARE_SOA_INDEX_COLUMN_FULL(Prong1Track, prong1Track, int, HfRedTrackBases, "_1"); //! Prong1 index
```

Comment:
Why are the suffixes needed?

Reply:
It's conflict with here: https://github.com/AliceO2Group/O2Physics/blob/10ad23c98ca010e01fa4a79c7d9deb06409ff15a/PWGHF/DataModel/CandidateReconstructionTables.h#L1926, I didn't want to modify the current lb base table, so i add the suffix for the reduced table

Reply:
How is it in conflict? None of these names exist in [CandidateReconstructionTables.h](https://github.com/AliceO2Group/O2Physics/blob/10ad23c98ca010e01fa4a79c7d9deb06409ff15a/PWGHF/DataModel/CandidateReconstructionTables.h#L1926)